### PR TITLE
try to use a different folder

### PR DESCRIPTION
**Pull request by darioizzo**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by darioizzo** [comment]
@conda-forge-admin, please rerender

**Comment by darioizzo** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,12 +21,12 @@ requirements:
...
 
   host:
+    - python
     - boost
     - eigen
     - obake-devel
     - mppp
     - audi {{ version }}
-    - python
     #- pybind11
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:30]: you will need pybind here, dont you?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,12 +21,12 @@ requirements:
     - ninja      # [win]
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:22]: I think you don't need ``clang``

File: recipe/meta.yaml (hunk 1 above)
Comment (darioizzo) [reply recipe/meta.yaml:30]: I am installing pybind11 HEAD when building the recipe as the currently released pybind has problems when compiled with the c++17 standard.

Later, when the new pybnd11 will be released and available in conda I can remove such a work around.

File: recipe/meta.yaml (hunk 2 above)
Comment (darioizzo) [reply recipe/meta.yaml:22]: Thanks for looking into this. I needthe compiler under windows to be clang-cl. I thought that this is the way to achieve it?

Comment (marcelotrevisani) [reply recipe/meta.yaml:22]: dont you need to skip ``{{ compiler('c') }}`` and ``{{ compiler('cxx') }}`` for windows then?

File: recipe/bld.bat [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
 cd pybind11
 mkdir build
 cd build
 cmake ^
     -G "Ninja" ^
     -DPYBIND11_TEST=NO ^
-    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
-    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
+    -DCMAKE_PREFIX_PATH=%PREFIX% ^
```
Comment (marcelotrevisani) [inline recipe/bld.bat:12]:
```suggestion
    -DCMAKE_PREFIX_PATH=%PREFIX% ^
    -DCMAKE_BUILD_TYPE=Release ^
```

File: recipe/bld.bat [hunk 4]
```diff
@@ -21,13 +19,12 @@ cmake ^
...
     -DCMAKE_CXX_COMPILER=clang-cl ^
     -DBoost_NO_BOOST_CMAKE=ON ^
-    -DCMAKE_BUILD_TYPE=Release ^
     -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
     -DAUDI_BUILD_TESTS=no ^
     -DAUDI_BUILD_AUDI=no ^
     -DAUDI_BUILD_PYAUDI=yes ^
-    -Dpybind11_DIR=%LIBRARY_PREFIX%/share/cmake/pybind11 ^
+    -Dpybind11_DIR=%PREFIX%/share/cmake/pybind11/ ^
```
Comment (marcelotrevisani) [inline recipe/bld.bat:28]:
```suggestion
    -Dpybind11_DIR=%PREFIX%/share/cmake/pybind11/ ^
    -DCMAKE_BUILD_TYPE=Release ^
```

File: recipe/meta.yaml (hunk 2 above)
Comment (darioizzo) [reply recipe/meta.yaml:22]: ok will do that, thanks.

File: recipe/bld.bat (hunk 3 above)
Comment (darioizzo) [reply recipe/bld.bat:12]: done thanks!

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Pull request merged**